### PR TITLE
Add if/else to check for empty resp['intents']

### PR DESCRIPTION
**Pull request by sv-github**
This fixes an indexError (seen during `python3 test_discovery.py`)

```
testing/evaluate_tests.py", line 284, in evaluate_entities_and_schema
  entities = most_likely_entities = resp["intents"][num_intents]["entities"]
```

### CODE REVIEW

File: testing/evaluate_tests.py [hunk 1]
```diff
@@ -281,12 +281,15 @@ def evaluate_entities_and_schema(
...
         logger.setLevel(logging.INFO)
 
-    entities = most_likely_entities = resp["intents"][num_intents]["entities"]
-    observed_entity_dict = {
-        # keep only the most likely hypothesis from Discovery -> first dict in list of dicts returned by Discovery
-        ent["label"]: ent["matches"][0]["value"]
-        for ent in entities  #most_likely_entities
-    }
+    if resp["intents"]:
+        entities = most_likely_entities = resp["intents"][num_intents]["entities"]
```
Comment (mgoldey) [inline testing/evaluate_tests.py]: Maybe inline this so it's fewer large if/else statements
```suggestion
        entities = most_likely_entities = resp["intents"][num_intents]["entities"] if "intents" in resp else []
```

Comment (sv-github) [reply testing/evaluate_tests.py]: :+1: all for the inline!

Comment (sv-github) [inline testing/evaluate_tests.py]: @tshastry @mgoldey 

`entities = most_likely_entities = resp["intents"][num_intents]["entities"] if resp["intents"] else []`
to
`entities  = resp["intents"][num_intents]["entities"] if resp["intents"] else []`

do we want to keep the commented variable `most_likely_entities` ?

Comment (mgoldey) [reply testing/evaluate_tests.py]: I vote no

**Pull request merged**